.github/workflows/upgrade.yaml

```diff
@@ -3,91 +3,39 @@ on:
   push:
     paths:
       - "manifest_staging/charts/**"
+      - ".github/workflows/upgrade.yaml"
```

Comment:
why do we add this under paths?

Reply:
same for pull_request

Reply:
This signals that the upgrade workflow should be run when the `upgrade.yaml` file itself gets updated. Otherwise we would have an untested test (that's fun to say)